game/scripts/vscripts/items/drums_of_endurance.lua

```diff
@@ -0,0 +1,259 @@
+LinkLuaModifier( "modifier_intrinsic_multiplexer", "modifiers/modifier_intrinsic_multiplexer.lua", LUA_MODIFIER_MOTION_NONE )
+LinkLuaModifier( "modifier_item_drums_of_endurance_oaa", "items/drums_of_endurance.lua", LUA_MODIFIER_MOTION_NONE )
+LinkLuaModifier( "modifier_item_drums_of_endurance_oaa_swiftness_aura", "items/drums_of_endurance.lua", LUA_MODIFIER_MOTION_NONE )
+LinkLuaModifier( "modifier_item_drums_of_endurance_oaa_swiftness_aura_effect", "items/drums_of_endurance.lua", LUA_MODIFIER_MOTION_NONE )
+LinkLuaModifier( "modifier_item_drums_of_endurance_oaa_active", "items/drums_of_endurance.lua", LUA_MODIFIER_MOTION_NONE )
+
+item_drums_of_endurance_oaa = class(ItemBaseClass)
+
+function item_drums_of_endurance_oaa:GetIntrinsicModifierName()
+  return "modifier_intrinsic_multiplexer"
+end
+
+function item_drums_of_endurance_oaa:GetIntrinsicModifierNames()
+  return {
+    "modifier_item_drums_of_endurance_oaa",
+    "modifier_item_drums_of_endurance_oaa_swiftness_aura",
+    "modifier_item_drums_of_endurance_oaa_swiftness_aura_effect"
```

Comment:
The effect should not be an intrinsic modifier.

Reply:
-Will remove the effect modifier